app/src/main/java/tech/ula/utils/AndroidUtility.kt

```diff
@@ -387,13 +389,42 @@ class LocalFileLocator(private val applicationFilesDir: String, private val reso
 }
 
 class DeviceDimensions {
-    private var width = 720
-    private var height = 1480
+    private var height = 720
+    private var width = 1480
 
-    fun getDeviceDimensions(windowManager: WindowManager, displayMetrics: DisplayMetrics) {
-        windowManager.defaultDisplay.getMetrics(displayMetrics)
-        width = displayMetrics.widthPixels
+    companion object {
+        const val portrait = "SCREEN_ORIENTATION_PORTRAIT"
+        const val landscape = "SCREEN_ORIENTATION_LANDSCAPE"
+    }
+
+    fun getDeviceDimensions(windowManager: WindowManager, displayMetrics: DisplayMetrics, orientation: Int) {
```

Comment:
I think this function could use a slightly different name. `get` to me implies it will return the dimensions.

Reply:
I changed `get` to `save`, I think that may be more clear